projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -44,6 +44,7 @@ public static function on_jetpack_feature_publicize_enabled() {
 		add_action( 'rest_api_init', array( new REST_Controller(), 'register_rest_routes' ) );
 		add_action( 'rest_api_init', array( new Social_Image_Generator\REST_Settings_Controller(), 'register_routes' ) );
 
+		self::init_social_image_generator();
```

Comment:
**JUST FOR TESTING PURPOSES!**

I will remove this before merging because we will use https://github.com/Automattic/jetpack/pull/29529 instead.